web/tests/api/utils.test.ts

```diff
@@ -47,15 +47,9 @@ describe("validateUrl()", () => {
   test("localhost prod", () => {
     expect(validateUrl(httpLocalhost, !isStaging)).toBeFalsy();
   });
-});
-
-describe("url js injection", () => {
-  it("without inject", () => {
-    expect(uriHasJS("http://test.com")).toBeFalsy();
-  });
 
-  it("with inject", () => {
-    expect(uriHasJS("javascript:alert('test')")).toBeTruthy();
-    expect(uriHasJS("javascript:;alert('test');")).toBeTruthy();
-  });
-});
+  test("insecure schema", () => {
+    expect(validateUrl("javascript:alert('test')", isStaging)).toBeFalsy();
+    expect(validateUrl("jaVasCript:;alert('test');", isStaging)).toBeFalsy();
+  })
+});
```

Comment:
I'm definitely not the expert on this repo, but wouldn't it be best to place this check under the `validateUrl()` suite?